content/reference/endpoints/accounts.md

```diff
@@ -0,0 +1,48 @@
+# Accounts
+
+1. TOC
+{:toc}
+
+<ul class="nav nav-pills pull-right" role="tablist">
+  <li class="disabled"><a>OAuth Scopes:</a></li>
+  <li class="active"><a href="#public" role="tab" data-toggle="pill">public</a></li>
+</ul>
+
+<div class="tab-content" markdown="1">
+  <div class="tab-pane active" id="public" markdown="1">
+### Parameters
+
+Name             | Type    | Read/Write | Description
+-----------------|---------|------------|------------
+id               | Integer | Read                                 | Account's id.
+-----------------|---------|------------|------------
+business_name    | Integer | Read                                 | Account's business name.
+-----------------|---------|------------|------------
+status           | String  | Read                                 | Account's status, list of statuses described in [enums section](/reference/enums/#account-statuses).
+-----------------|---------|------------|------------
+created_at       | [Time](/reference/enums#formats)  | Read       | Account's create time.
+-----------------|---------|------------|------------
+updated_at       | [Time](/reference/enums#formats)  | Read       | Account's update time.
+{: class="table table-bordered"}
+  </div>
+</div>
+
+## List accounts
+
+List all accounts for given account(s).
```

Comment:
Something feels wrong here :P What about:

`List all authorized accounts. Only the [Client Credentials Flow](/reference/authorization/#client-credentials-flow) currently allow you to retrieve multiple accounts at ones.`